pkg/util/iptables/iptables.go

```diff
@@ -118,9 +118,11 @@ const NoFlushTables FlushFlag = false
 // (test whether a rule exists).
 const MinCheckVersion = "1.4.11"
 
-// Minimum iptables versions supporting the -w and -w2 flags
+// Minimum iptables versions supporting the -w and -w<seconds> flags
 const MinWaitVersion = "1.4.20"
-const MinWait2Version = "1.4.22"
+const MinWaitSecondsVersion = "1.4.22"
```

Comment:
None of these variables should be named "Min".  Min is for the versions - the minimum version required.

Reply:
Fixed. Thanks.